### ISSUE TITLE
[Feature] Departure is read when double tapped

### ISSUE THREAD

**Issue by hildeor** [AtB-AS/mittatb-app#552]
## Origin
_Links to received feedback, user research or other findings._

While in voiceover mode (iOS):
When double tapping (as in click button), the departure is read out by the voiceover, and is then interrupted by the new modal that appears (details). 

Additional info: The element is selected (and read out loud as intended), but after being activated, the element re-gains focus and is being read out loud again.

By further investigation, this observed issue seem to be a symptom of an issue with setting focus after navigating/opening modal on iOS. The activated element in the previous view regains focus upon opening modal. I

## Motivation
_A short description of what user needs or business goals this feature will solve._

## Hypotheses and assumptions
_A list of hypotheses and assumptions we have made about the user or the proposed solution._

## Proposed solution
_A coarse description of a proposed solution, that may include wireframes and graphic design._

This is a symptom of a greater problem, which will be described in a separate issue

**Comment by hildeor** [comment]
Unable to reproduce this now.
